src/main/java/it/gov/pagopa/arc/exception/ArcExceptionHandler.java

```diff
@@ -23,11 +23,6 @@ public ResponseEntity<ErrorDTO> handleBizEventsInvocationException(RuntimeExcept
         return handleArcErrorException(ex, request, HttpStatus.INTERNAL_SERVER_ERROR, ErrorDTO.ErrorEnum.GENERIC_ERROR);
     }
 
-    @ExceptionHandler(BizEventsTransactionNotFoundException.class)
-    public ResponseEntity<ErrorDTO> handleBizEventsTransactionNotFoundException(RuntimeException ex, HttpServletRequest request){
-        return handleArcErrorException(ex, request, HttpStatus.NOT_FOUND, ErrorDTO.ErrorEnum.TRANSACTION_NOT_FOUND_ERROR);
```

Comment:
Remove the ErrorEnum "transaction_not_found_error" from the swagger

Reply:
removed